### PR TITLE
appnexus bid adapter - add support for addtlConsent

### PR DESCRIPTION
**Pull request by jsnellbaker**
## Type of change

- [x] Feature

## Description of change
Adds support for the GDPR addtlConsent field in the appnexus bid adapter.

### CODE REVIEW

**Review by patmmccann — APPROVED** [review]

**Comment by patmmccann** [comment]
@jsnellbaker this looks good to merge to me,  up to you if your want to wait for fawke or if you prefer to make today's release,  expected to be the last release before 5.0

**Review by Fawke — APPROVED** [review]

**Pull request merged**
